test_rules_scala.sh

```diff
@@ -803,11 +803,11 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 $runner bazel build test/...
-$runner bazel build "test/... --all_incompatible_changes"
+#$runner bazel build "test/... --all_incompatible_changes"
```

Comment:
should we have an issue about reviving it? (will try to take a closer look later)

Reply:
@ittaiz Totally -- hopefully we can get it back in soon. I took care of a bunch of them, but lost steam when it came to doing the create_provider -> JavaInfo, it looked a little involved since it seems like we should be separating out the transitive compile time jars from the current target's compile jars for the newer API.